mp4parse/src/lib.rs

```diff
@@ -2352,6 +2371,7 @@ fn read_moov<T: Read>(f: &mut BMFFBox<T>, context: Option<MediaContext>) -> Resu
         mut mvex,
         mut psshs,
         mut userdata,
+        metadata,
```

Comment:
```suggestion
        #[cfg(feature = "meta-xml")]
        metadata,
```
Required based on putting `MediaContext.metadata` behind the feature flag